scss/os/_slicktree.scss

```diff
@@ -62,7 +62,9 @@
 
 // Node Icons
 .c-node-icons {
```

Comment:
There is one more `.fa` specific style on line 12 of this file we'll want to update.